rts/pom.xml

```diff
@@ -31,9 +31,9 @@ Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
   <url>http://www.jboss.org/jbosstm</url>
  
   <properties>
-    <version.wildfly-swarm>2017.2.0</version.wildfly-swarm>
+    <version.wildfly-swarm>2.3.0.Final</version.wildfly-swarm>
```

Comment:
possibly renamed for `thorntail`?

Reply:
They're just names and besides the name change could brake any existing scripts/notes that use the property. Why do you feel it's important?

Reply:
good point, we shouldn;t just drop these externally facing property values as they could be in other scripts (e.g. user ones). I think it would be a good follow up task though

Reply:
I think we should rename it as we are doing refactoring for the swarm to thorntail and we should not contribute to the technical debt for the project and for the bad understandability.

Reply:
I agree, for this one it seems definitely viable to change to thorntail as the whole GAV has changed so it is very unlikely that a -Dversion.wildfly-swarm=2017.2.0 or similar would work with the new artifact we are using